<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandPrompt</class>
 <widget class="QDialog" name="CommandPrompt">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectPushButton">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectPushButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="commandPromptTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="commandLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataFormatComboBox">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
         <property name="currentText">
          <string> String</string>
         </property>
         <item>
          <property name="text">
           <string> String</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string> Binary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string> Hex</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendPushButton">
         <property name="text">
          <string>Send</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>commandLineEdit</tabstop>
  <tabstop>sendPushButton</tabstop>
  <tabstop>dataFormatComboBox</tabstop>
  <tabstop>commandPromptTextEdit</tabstop>
  <tabstop>disconnectPushButton</tabstop>
  <tabstop>connectPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
